server/config/base.js

```diff
@@ -138,7 +138,18 @@ module.exports = {
             propertyToMatch: 'url2',
             fieldDescription: 'test trait'
         }
-    ]
+    ],
+    
+    tagValuesTransformMap: {
+        success: {
+            type: 'boolean'
+        },
+        url2: {
+            type: 'link',
+            urlTemplate: 'https://www.google.com/search?q=#{value}',
+            textTemplate: '[Google] #{value}'
```

Comment:
Since the transform type is `link`, more configuration is necessary. The `linkTransform` returns a link jsx with a custom url and a custom displayed text. `urlTemplate` is the pattern that will be used to generate the url. The `textTemplate` is the pattern to generate the text.  In both templates, `#{value}` in the strings will be replaced with the value of the tag. Both parameters are necessary for this type of transform.